### PR TITLE
K8SPXC-317 Made imagePullPolicy configurable

### PR DESCRIPTION
**Pull request by nonemax**
[K8SPXC-317](https://jira.percona.com/browse/K8SPXC-317)

### CODE REVIEW

**Review by delgod — APPROVED** [review]

File: deploy/cr.yaml [hunk 1]
```diff
@@ -48,6 +48,7 @@ spec:
 #      runAsGroup: 1001
 #      supplementalGroups: [1001]
 #    serviceAccountName: percona-xtradb-cluster-operator-workload
+#    imagePullPolicy: Always
```
Comment (delgod) [inline deploy/cr.yaml:51]: can we add changing of this field to some exiting test?
e.g. `limits` test

**Review by delgod — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| init-deploy | passed |
| storage | passed |
| scaling | passed |
| big-data | passed |
| self-healing | passed |
| restore-to-encrypted-cluster | passed |
| scaling-proxysql | passed |
| limits | passed |
| monitoring | passed |
| upgrade | passed |
| self-healing-advanced | passed |
| upgrade-consistency | passed |
| demand-backup | passed |
| security-context | passed |
| monitoring-2-0 | passed |
| operator-self-healing | passed |
| affinity | passed |
| scheduled-backup | failed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| storage | passed |
| big-data | passed |
| recreate | passed |
| init-deploy | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| limits | passed |
| restore-to-encrypted-cluster | passed |
| upgrade | passed |
| monitoring | passed |
| self-healing-advanced | passed |
| upgrade-consistency | passed |
| security-context | passed |
| demand-backup | passed |
| operator-self-healing | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| scheduled-backup | failed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup-encrypted-with-tls | failed |

File: deploy/cr.yaml (hunk 1 above)
Comment (nonemax) [reply deploy/cr.yaml:51]: added https://github.com/percona/percona-xtradb-cluster-operator/pull/545/commits/258fd93e2579a53930e2ef947768a8fbd9b823e0

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| big-data | passed |
| init-deploy | passed |
| recreate | passed |
| scaling | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| limits | passed |
| restore-to-encrypted-cluster | passed |
| upgrade | passed |
| monitoring | passed |
| upgrade-consistency | passed |
| monitoring-2-0 | passed |
| security-context | passed |
| demand-backup | passed |
| self-healing-advanced | passed |
| affinity | passed |
| one-pod | passed |
| scheduled-backup | failed |
| operator-self-healing | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup-encrypted-with-tls | passed |

**Review by delgod — APPROVED** [review]

**Review by qJkee — DISMISSED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| recreate | passed |
| scaling | passed |
| init-deploy | passed |
| big-data | passed |
| self-healing | passed |
| limits | passed |
| restore-to-encrypted-cluster | passed |
| scaling-proxysql | passed |
| monitoring | passed |
| self-healing-advanced | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| security-context | passed |
| affinity | passed |
| one-pod | passed |
| operator-self-healing | passed |
| scheduled-backup | failed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup-encrypted-with-tls | failed |

**Review by delgod — APPROVED** [review]

**Pull request merged**